docs/modules/operation/pages/alarms/configuring-alarms.adoc

```diff
@@ -144,6 +150,7 @@ org.opennms.alarmd.newIfClearedAlarmExists = true
 ----
 
 Now, with this property set, when a repeat incident occurs and the current state of the Alarm tracking the problem is "Cleared", instead of restating the current Alarm to it's default severity and incrementing the counter, a new instance of the Alarm will be created.
```

Comment:
```suggestion
Now, with this property set, when a repeat incident occurs and the current state of the alarm tracking the problem is "Cleared", instead of restating the current alarm to its default severity and incrementing the counter, a new instance of the alarm will be created.
```